src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerArgumentCollection.java

```diff
@@ -39,10 +43,36 @@
     @Argument(fullName = StandardArgumentDefinitions.SAMPLE_NAME_LONG_NAME, shortName = StandardArgumentDefinitions.SAMPLE_ALIAS_SHORT_NAME, doc = "Name of single sample to use from a multi-sample bam", optional = true)
     public String sampleNameToUse = null;
 
+    /**
+     * rsIDs from this file are used to populate the ID column of the output. Also, the DB INFO flag will be set when appropriate.
+     * dbSNP is not used in any way for the calculations themselves.
+     */
+    @ArgumentCollection
+    public DbsnpArgumentCollection dbsnp = new DbsnpArgumentCollection();
+
     // -----------------------------------------------------------------------------------------------
     // general advanced arguments to control haplotype caller behavior
     // -----------------------------------------------------------------------------------------------
 
+    /**
+     * If a call overlaps with a record from the provided comp track, the INFO field will be annotated
+     * as such in the output with the track name (e.g. -comp:FOO will have 'FOO' in the INFO field). Records that are
+     * filtered in the comp track will be ignored. Note that 'dbSNP' has been special-cased (see the --dbsnp argument).
+     */
+    @Advanced
+    @Argument(fullName = "comp", shortName = "comp", doc = "Comparison VCF file(s)", optional = true)
+    public List<FeatureInput<VariantContext>> comps = Collections.emptyList();
+
+    /**
+     * The reference confidence mode makes it possible to emit a per-bp or summarized confidence estimate for a site being strictly homozygous-reference.
+     * See http://www.broadinstitute.org/gatk/guide/article?id=2940 for more details of how this works.
+     * Note that if you set -ERC GVCF, you also need to set -variant_index_type LINEAR and -variant_index_parameter 128000 (with those exact values!).
+     * This requirement is a temporary workaround for an issue with index compression.
```

Comment:
While you're here, would you mind deleting this obsolete section of the docs:

```
Note that if you set -ERC GVCF, you also need to set -variant_index_type LINEAR and -variant_index_parameter 128000 (with those exact values!).
This requirement is a temporary workaround for an issue with index compression.
```

Reply:
done